tests/scripts/all.sh

```diff
@@ -1456,13 +1456,28 @@ component_test_tls1_2_ecjpake_compatibility() {
     make -C programs test/udp_proxy test/query_compile_time_config
 
     msg "test: server w/o USE_PSA - client w/ USE_PSA"
-    P_SRV=../s2_no_use_psa tests/ssl-opt.sh -f ECJPAKE
+    P_SRV=../s2_no_use_psa tests/ssl-opt.sh -f ECJPAKE -e ECJPAKE_OPAQUE_PW
     msg "test: client w/o USE_PSA - server w/ USE_PSA"
-    P_CLI=../c2_no_use_psa tests/ssl-opt.sh -f ECJPAKE
+    P_CLI=../c2_no_use_psa tests/ssl-opt.sh -f ECJPAKE -e ECJPAKE_OPAQUE_PW
 
     rm s2_no_use_psa c2_no_use_psa
 }
 
+# Opaque password testing requires a separate test with repect to
+# "test_tls1_2_ecjpake_compatibility". In that case there's a mix of PSA and
+# MbedTLS based implementations of EC-JPAKE which makes it difficult to parse
+# proper strings during the test. As a consequence here we just build the
+# PSA variant for both client and server.
+component_test_tls1_2_ecjpake_opaque_password() {
```

Comment:
I don't think that component is needed. We already have a number of components that build with `config.py full` (which in particular enables `USE_PSA`) and run `ssl-opt.sh` in full. For example, `component_test_full_cmake_gcc_asan`.